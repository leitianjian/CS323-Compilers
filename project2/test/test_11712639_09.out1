Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        StructSpecifier (1)
          STRUCT
          ID: st
          LC
          DefList (2)
            Def (2)
              Specifier (2)
                TYPE: int
              DecList (2)
                Dec (2)
                  VarDec (2)
                    ID: a
                COMMA
                DecList (2)
                  Dec (2)
                    VarDec (2)
                      VarDec (2)
                        ID: b
                      LB
                      INT: 2
                      RB
              SEMI
          RC
      SEMI
    ExtDefList (5)
      ExtDef (5)
        Specifier (5)
          StructSpecifier (5)
            STRUCT
            ID: st
            LC
            DefList (6)
              Def (6)
                Specifier (6)
                  TYPE: int
                DecList (6)
                  Dec (6)
                    VarDec (6)
                      ID: c
                SEMI
              DefList (7)
                Def (7)
                  Specifier (7)
                    TYPE: char
                  DecList (7)
                    Dec (7)
                      VarDec (7)
                        ID: c
                  SEMI
            RC
        SEMI
