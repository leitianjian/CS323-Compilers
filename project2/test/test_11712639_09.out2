Error type 15 at Line 5: redefine the same structure type
Error type 3 at Line 7: c variable is redefined in the same scope
