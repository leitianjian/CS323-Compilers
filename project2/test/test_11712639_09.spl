struct st{
    int a, b[2];
};

struct st{
    int c;
    char c;
};