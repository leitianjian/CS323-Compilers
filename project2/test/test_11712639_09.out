Error type 15 at line 5: redefine structure type st
Error type 3 at line 7: c variable is redefined in the same scope
